app/models/member.rb

```diff
@@ -57,6 +57,14 @@ def not_employed?
     employment_status == "not_employed"
   end
 
+  def female?
+    sex == "female"
```

Comment:
should we add a validation on `Member` that `sex` is one of these two values? right now that validation happens at the step level and I'd be worried about it being `Female` instead of `female` or something like that

Reply:
Yeah, we should. I think I've seen it capitalized in other places.